test/scripts/gh-actions/build-server-runtimes-huggingface-server.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Copyright 2022 The KServe Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The script is used to build KServe Huggingface Server images.
+
+# TODO: Implement selective building and tag replacement based on modified code.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+IFS=,
+
+echo "Github SHA ${GITHUB_SHA}"
+# Predictor runtime server images for huggingface server
+HUGGINGFACE_IMG_TAG=${DOCKER_REPO}/${HUGGINGFACE_IMG}:${GITHUB_SHA}-gpu
+HUGGINGFACE_CPU_OPENVINO_IMG_TAG=${DOCKER_REPO}/${HUGGINGFACE_IMG}:${GITHUB_SHA}
+types=("$1")
+
+pushd python >/dev/null
+  if [[ " ${types[*]} " =~ "predictor" ]]; then
+    echo "Building Huggingface image"
+    docker buildx build -t "${HUGGINGFACE_IMG_TAG}" -f huggingface_server.Dockerfile \
```

Comment:
@yuzisun I think we don't need the gpu image for e2e tests for now. We can test both vllm and huggingface backend using the cpu image. WDYT?

Reply:
yeah right, I will remove gpu image and keep one job only then